tests/idris2/coverage010/expected

```diff
@@ -1,8 +1,9 @@
 1/1: Building casetot (casetot.idr)
 Error: main is not covering.
 
-casetot.idr:12:1--13:5
+casetot.idr:12:1--12:13
+    |
  12 | main : IO ()
- 13 | main = do
+    | ^^^^^^^^^^^^
 
 Calls non covering function Main.case block in case block in main
```

Comment:
Any chance we can have the error point at the faulty case rather than
the toplevel function enclosing it?